### PR TITLE
proposed changes after following trainings

### PR DESCRIPTION
**Pull request by kafetzakid**

### CODE REVIEW

**Comment by jefscheepers** [comment]
@kafetzakid I have two comments (see above). Apart from these, your changes are very welcome.

**Review by jefscheepers — COMMENTED** [review]
See comments

File: 03_ManGO_Portal_Handson_User-Training.md [hunk 1]
```diff
@@ -321,12 +321,12 @@ In this exercise, you will use the files corpus1.txt, corpus2.txt and corpus3.tx
...
-    - Attribute 'language' and value 'french' to corpus2.txt  
-    - Attribute 'language' and value 'latin' to corpus3.txt  
-- Oops, we made a mistake! Open the file corpus2.txt, and look what the language is.  
+    - Attribute name 'language' and value 'dutch' to corpus1.txt  
+    - Attribute name 'language' and value 'french' to corpus2.txt  
+    - Attribute name 'language' and value 'latin' to corpus3.txt  
+- Oops, we made a mistake! Open the file corpus2.txt locally, and look what the language is.  
   Overwrite the current AVU with one with the correct value.  
-- Go to the 'search' tab and search for all files with Metadata Attribute Name 'language' and Metadata Attribute Value 'latin'.  
+- Go to the 'search' tab and search for all objects with Metadata Attribute Name 'language' and Metadata Attribute Value 'latin'.
```
[the remark this answers is not in the document]
Comment (kafetzakid) [reply 03_ManGO_Portal_Handson_User-Training.md:329]: Indeed, I modified accordingly.

File: 05_iCommands_Handson_User-Training.md [hunk 2]
```diff
@@ -655,7 +655,7 @@ iquest "SELECT DATA_NAME where META_DATA_ATTR_NAME = 'language' and META_DATA_AT
 imeta qu -d language = dutch
 ```
 ```sh
-imeta addw 
+imeta addw -d corpus_.txt type text
```
[the remark this answers is not in the document]
Comment (kafetzakid) [reply 05_iCommands_Handson_User-Training.md:658]: Okay, I modified accordingly.

**Review by jefscheepers — APPROVED** [review]
Perfect, than you for your changes!

**Review by jefscheepers — APPROVED** [review]
Looks nice, this seems ready to be merged.

**Pull request merged**